include/hirediscommand.h

```diff
@@ -195,15 +196,15 @@ namespace RedisCluster
         
         redisReply* process()
         {
-            redisReply *reply;
+            redisReply *reply = nullptr;
             typename Cluster::SlotConnection con = cluster_p_->getConnection( key_ );
             typename Cluster::HostConnection hcon = { "", NULL };
             string host, port;
-            
+
             reply = processHiredisCommand( con.second );
+            HiredisProcess::checkCritical(reply, false, "", con.second);
             cluster_p_->releaseConnection( con );
```

Comment:
i'll accept and later i'll change the order of HiredisProcess::checkCritical and cluster_p_->releaseConnection invocation order